rules/DowngradePhp71/Rector/Array_/SymmetricArrayDestructuringToListRector.php

```diff
@@ -66,13 +63,8 @@ public function refactor(Node $node): ?Node
         return null;
     }
 
-    private function processToList(Array_ $array): FuncCall
+    private function processToList(Array_ $array): List_
     {
-        $args = [];
-        foreach ($array->items as $arrayItem) {
-            $args[] = $arrayItem instanceof ArrayItem ? new Arg($arrayItem->value) : null;
-        }
-
-        return new FuncCall(new Name('list'), $args);
+        return new List_($array->items);
```

Comment:
test case for empty array item may be needed to avoid regression; ref https://3v4l.org/WoXBE

Reply:
There is already a test case for `[, $e1] = ...` in [.../SymmetricArrayDestructuringToListRector/Fixture/include_empty.php.inc](https://github.com/rectorphp/rector-downgrade-php/blob/main/rules-tests/DowngradePhp71/Rector/Array_/SymmetricArrayDestructuringToListRector/Fixture/include_empty.php.inc)

Or a check for `[] = ...`? https://getrector.org/demo/fcf6d4e1-de3d-44ca-a6f5-7aeddb488e9a But that's invalid code anyway: https://3v4l.org/AZAv4

Reply:
I see 👍 , mergeable then